apps/indexer/src/api/controller/proposals.ts

```diff
@@ -44,7 +50,17 @@ export function proposals(app: Hono, service: ProposalsService) {
         fromDate,
       });
 
-      return context.json(result.map(ProposalMapper.toApi));
+      const quorums = await Promise.all(
+        result.map((p) => client.getQuorum(p.id)),
+      );
+
+      const votingDelay = await service.getVotingDelay();
+
+      return context.json(
+        result.map((p, index) =>
+          ProposalMapper.toApi(p, quorums[index]!, blockTime, votingDelay),
```

Comment:
Is votingDelay necessary? I mean, can it be a proposal's property? Isnt it more like a dao's property?